self-host/self-host-react-example/src/App.tsx

```diff
@@ -171,8 +230,9 @@ function App() {
 							'Private Key: ' + reconstructedKey.privKey.toString("hex"),
 						);
 					}
-					const shareStore = await tKey.generateNewShare();
-					await (tKey.modules.webStorage as any).storeDeviceShare(shareStore.newShareStores[1]);
+					const newShare = await tKey.generateNewShare();
+					const shareStore = await tKey.outputShareStore(newShare.newShareIndex);
+					await (tKey.modules.webStorage as any).storeDeviceShare(shareStore);
```

Comment:
This need to fix in main branch

Reply:
This makes sense. We were just trying to use the latest share